docs/developer/channels.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Channels
+sidebar_label: Channels
+---
+
+## Introduction
+
+Since version 3.0, Saleor supports multichannel. This feature allows the shop owner for configuring many aspects of the store depending on a channel.
```

Comment:
```suggestion
Since version 3.0, Saleor supports multiple sales channels. This feature allows the shop owner to configure many aspects of the store depending on the channel used.
```